code/game/objects/items/surgery_tray.dm

```diff
@@ -157,59 +168,89 @@
 
 /obj/item/surgery_tray/deployed
 	is_portable = FALSE
+	trace_if_mapped_in = FALSE
```

Comment:
I think this is unecessarily protective. Mappers should be able to choose to map in a tray (or a toolbox) if they so desire.

Reply:
ok